frontend/src/containers/DocumentList.js

```diff
@@ -155,6 +154,7 @@ class DocumentListContainer extends Component {
      * middleware reacting to route changes and reducers
      */
     if (
+      nextProps.viewId !== nextProps.queryViewId ||
```

Comment:
please add a comment when this is the case so we won't have to figure it out in the future

Reply:
Done. Thanks